src/main.cpp

```diff
@@ -1046,6 +1046,8 @@ int main(int argc, char **args)
   programState.activeViewportIndex = 0;
   programState.pActiveViewport = &programState.pViewports[programState.activeViewportIndex];
 
+  programState.flythroughExportCancel = false;
```

Comment:
This should already be the case?

Reply:
no this is the initialization part of main, so it hasn't been assigned yet

Reply:
As discussed on teams, the `programState` is initialised with `{}` so this is already `false`.